src/App.tsx

```diff
@@ -156,6 +154,13 @@ function App() {
         <Route path="/:language/Insights">
           <Insights />
         </Route>
+        <Route path="/:language/Unity">
+          <Explore initialFilters={(() => {
+            const initialFilters = getEmptyFilters();
+            initialFilters.unity_aligned_only = true;
```

Comment:
Didn't know you could update `const` - but read more about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const. JavaScript is weird

Reply:
yeah since we are not technically updating the reference we can update the const like this. this is a bit sketchy tho, I will change it to a `let` so what we're doing here is more apparent